BuoyTest/src/components/cocktails/CocktailDetails.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { Stylesheet, View, Text, Image } from 'react-native';
+import AppService from '../../providers/app/AppService';
+import { NavigationActions } from 'react-navigation';
+import CocktailView from './details/CocktailView';
+
+class CocktailDetails extends Component {
+  state = {
+    cocktail: null
+  };
+  componentDidMount() {
+    this.loadCocktailDetails();
+  }
+  loadCocktailDetails() {
+      AppService.getCocktailDetails(
+          this.props.navigation.getParam('cocktailId', ''),
+          cocktail => {
+              this.setState({ cocktail: cocktail });
+          },
+          err => {
+              const modalProps = {
+                  type: 'error',
+                  buttonOk: 'OK',
+                  message: `Load cocktail details error:${err}`
+              };
+          }
+      );
+  }
+  render() {
+    return (
+      <View style={{ flex: 1 }}>
```

Comment:
There is possible to write this inline style, but is not recommended. You are creating a new object in memory each time this render is executed. The better way is to use all styles through Stylesheet.create

Reply:
Updated!